tests/components/zwave_mqtt/test_scenes.py

```diff
@@ -0,0 +1,100 @@
+"""Test Z-Wave (central) Scenes."""
+import asyncio
+import json
+from unittest.mock import Mock
+
+from .common import async_capture_events, setup_zwave
+
+
+async def test_scenes(hass, generic_data, sent_messages):
+    """Test setting up config entry."""
+
+    receive_message = await setup_zwave(hass, fixture=generic_data)
+    events = async_capture_events(hass, "zwave_mqtt.scene_activated")
+
+    # Publish fake scene event on mqtt
+    receive_message(
+        Mock(
```

Comment:
Didn't we have a fake `MQTTMessage` defined earlier for this use case ?

Reply:
We have the real message class:
https://github.com/home-assistant/core/blob/799d98eaf055db82265bcea18cb4f584c15ab5ff/homeassistant/components/mqtt/models.py#L10-L19

Reply:
Or do you mean the one we have defined here:
https://github.com/home-assistant/core/pull/34987/files#diff-2563762a93a51606add0ccad6831ffc8R56-R70

Reply:
yeah I meant that one. I guess just a helper class so that not every test needs to include a `json.dumps`

Reply:
Commit done.